grammar/pat.lyg

```diff
@@ -1,18 +1,30 @@
 Pat =
   | Wild:"_"
+  | Rest:".."
   | Literal:{ minus:"-"? lit:LITERAL }
   // unstable(exclusive_range_pattern):
-  | Range:{ start:PatRangeValue ".." end:PatRangeValue }
-  | RangeInclusive:{ start:PatRangeValue { "..." | "..=" } end:PatRangeValue }
+  | Range:{
+    | start:PatRangeValue ".." end:PatRangeValue
+    // unstable(half_open_range_patterns):
+    | start:PatRangeValue ".." | ".." end:PatRangeValue
+  }
+  | RangeInclusive:{
+    | start:PatRangeValue { "..." | "..=" } end:PatRangeValue
```

Comment:
```suggestion
    | start:PatRangeValue "..=" end:PatRangeValue
```

Reply:
We can't backwards-compatibly remove the `...` syntax, can we? I thought it just warns.

Reply:
Ah woops; there's a start here; disgregard this, but the other comment still applies.